packages/flutter/lib/src/material/menu_bar_theme.dart

```diff
@@ -0,0 +1,333 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'material_state.dart';
+import 'menu_bar.dart';
+import 'theme.dart';
+
+/// Defines the visual properties of [MenuBar], [MenuBarMenu] and
+/// [MenuBarButton] widgets.
+///
+/// Descendant widgets obtain the current [MenuBarThemeData] object
+/// using `MenuBarTheme.of(context)`. Instances of
+/// [MenuBarThemeData] can be customized with
+/// [MenuBarThemeData.copyWith].
+///
+/// Typically, a [MenuBarThemeData] is specified as part of the
+/// overall [Theme] with [ThemeData.menuBarTheme]. Otherwise,
+/// [MenuBarTheme] can be used to configure its own widget subtree.
+///
+/// All [MenuBarThemeData] properties are `null` by default.
+/// If any of these properties are null, the menu bar will provide its own
+/// defaults.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+@immutable
+class MenuBarThemeData with Diagnosticable {
+  /// Creates the set of properties used to configure [MenuBarTheme].
+  const MenuBarThemeData({
+    this.barHeight,
+    this.barPadding,
+    this.barBackgroundColor,
+    this.barElevation,
+    this.menuBackgroundColor,
+    this.menuElevation,
+    this.menuShape,
+    this.menuPadding,
+    this.itemBackgroundColor,
+    this.itemForegroundColor,
+    this.itemOverlayColor,
+    this.itemTextStyle,
+    this.itemPadding,
+    this.itemShape,
+  });
+
+  /// The minimum height of the menu bar.
+  final double? barHeight;
```

Comment:
Is this really a minimum or is the bar's height fixed. If it's a minimum then maybe `minBarHeight`.

Generally speaking, theme properties should say: "Overrides the default [MenuBar.minimumHeight]" since the default and the states the property is resolved against will be documented by the MenuBar property.